src/applications/disability-benefits/all-claims/pages/uploadPersonalPtsdDocuments.js

```diff
@@ -1,33 +1,23 @@
 import { uploadDescription } from '../content/fileUploadDescriptions';
 import { ptsd781aNameTitle } from '../content/ptsdClassification';
 import { ancillaryFormUploadUi } from '../utils';
+import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
```

Comment:
Is this schema in sync with the one in the vets-website repo? It seems a little concerning to use both at the same time.